### ISSUE TITLE
Support automatic browser reload

### ISSUE THREAD

**Issue by cfjedimaster** [SimbCo/httpster#34]
So this would be an edge feature maybe, but perhaps support automatic browser reloading when the file changes. I've gotten used to it with other platforms, like Eleventy, and it would rock if httpster supported it. 

Note it does NOT work unless you've got a full HTML page (html, head, body) but I'd be fine with that.

**Comment by simb** [comment]
I agree that could be kinda hot.  I was planning to do an update to bring the deps upto date. Maybe this is something we can slip in.

The trick (I think) is that we need to inject some JS but only into the main HTML page that you are loading.  Or maybe it would get injected into any html page loaded by httpster?

Thoughts on how you would like this to work? @cfjedimaster

**Comment by cfjedimaster** [comment]
I'd say if the filename ends in .html, try the injection, and as I said, if I write an HTML file w/o "proper" head/body, I'm ok with it not working.

And obviously, make it optional and perhaps off by default.
